components/card-become-council.tsx

```diff
@@ -0,0 +1,50 @@
+import React, { useState } from "react";
+
+import { useTranslation } from "next-i18next";
+import Link from "next/link";
+
+import ArrowRight from "assets/icons/arrow-right";
+import CloseIcon from "assets/icons/close-icon";
+
+import { formatNumberToNScale } from "helpers/formatNumber";
+import { highlightText } from "helpers/string";
+
+import useNetwork from "x-hooks/use-network";
+
+import Button from "./button";
+
+export default function CardBecomeCouncil() {
+  const { t } = useTranslation("council");
+  const [show, setShow] = useState<boolean>(true)
+  const { network: activeNetwork, getURLWithNetwork } = useNetwork();
+
+  if(!show) return null;
+
+  return (
+    <div className="become-council p-2 ps-4 pb-3">
+      <div className="d-flex d-flex justify-content-end">
+        <Button transparent className="card-close-button p-1" onClick={() => setShow(false)}>
+          <CloseIcon
+            className="text-purple"
+          />
+        </Button>
+      </div>
+      <h4 className="h4 pb-2">{t("become-council")}</h4>
+      <div className="text-gray pe-3 pb-2">
+      <span
+              dangerouslySetInnerHTML={{
```

Comment:
?!?! _why_ ?

Reply:
A solution to style a specific name within a text without having to split it in council.json